main.go

```diff
@@ -168,30 +194,38 @@ func main() {
 			os.Exit(2)
 		}
 		auth.template = string(tmpl)
-		sshConfig.KeyboardInteractiveCallback = auth.keyboardInteractiveCallback
+		server.KeyboardInteractiveHandler = auth.keyboardInteractiveHandler
 	}
 
-	if viper.GetString("sentry-dsn") != "" {
-		if err := raven.SetDSN(viper.GetString("sentry-dsn")); err != nil {
-			log.Fatal("Sentry initialisation failed", err)
+	server.ConnCallback = func(ctx ssh.Context, conn net.Conn) net.Conn {
+		if verbosity >= 2 {
```

Comment:
It would be nice to use an enum or const with the matching level name rather than the magic `2`